ui/pages/settings/settings-tab/settings-tab.component.js

```diff
@@ -334,7 +334,7 @@ export default class SettingsTab extends PureComponent {
                   htmlFor="native-primary-currency"
                   className="settings-tab__radio-label"
                 >
-                  {nativeCurrency}
+                  {t('crypto')}
```

Comment:
Spoke with Albert today and he decided we should not make this change.  Since we know the symbol of the native currency (ETH, MATIC, BNB, etc), it's better to keeep the more specific symbol.